packages/core-utils/src/events.ts

```diff
@@ -1,11 +1,5 @@
 import { ethers } from 'ethers'
 
-export interface EventArgsAddressSet {
```

Comment:
Actually I would keep these for now. We might end up needing them in the DTL/data indexer soon.

Reply:
if autogenerated by the typechain, agree that it makes sense to remove here.

Reply:
Hrm true. Let me take a look at this.

Reply:
Doesn't seem like typechain exports this in the way we want it to. Better to leave this for another issue IMO.

Reply:
got it will add back